### PR TITLE
Create new example and update genomic filtering

### PR DESCRIPTION
**Pull request by emhug**

### CODE REVIEW

File: NHLBI_BioData_Catalyst/R/4_Genomic_Queries.ipynb [hunk 1]
```diff
@@ -486,13 +556,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Criteria 2: Participants with variants of high or moderate severity on *SERPINA1*"
+    "### Criteria 2: Participants with a missense variant on *SERPINA1*\n",
+    "\n",
+    "A filter for the *SERPINA1* gene can be added ina  way similar to the first query shown above."
```
Comment (simranmakwana) [inline NHLBI_BioData_Catalyst/R/4_Genomic_Queries.ipynb:561]: in a

File: NHLBI_BioData_Catalyst/R/4_Genomic_Queries.ipynb [hunk 2]
```diff
@@ -501,26 +577,43 @@
     "                                 categories = list('SERPINA1'))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can take a look a tthe variant consequences available for filtering to confirm that missense variant is a filtering option."
```
Comment (simranmakwana) [inline NHLBI_BioData_Catalyst/R/4_Genomic_Queries.ipynb:584]: the

File: NHLBI_BioData_Catalyst/Rstudio/4_Genomic_Queries.Rmd [hunk 3]
```diff
@@ -51,31 +51,33 @@ session <- picsure::bdc.setResource(session = session,  resourceName = "AUTH")
...
 
 First we will create a new query instance.
 
 ```{r}
 my_query <- picsure::bdc.newQuery(session) # Initialize a new query
 ```
 
-### Find all SAGE variables
+### Find all TOPMed DCC Harmonized variables
 We can search for variables related to our search query using the `bdc::find.in.dictionary` function.
```
Comment (simranmakwana) [inline NHLBI_BioData_Catalyst/Rstudio/4_Genomic_Queries.Rmd:65]: I don't think this is the function used any more, isn't this searchPicsure now?

File: NHLBI_BioData_Catalyst/R/4_Genomic_Queries.ipynb [hunk 4]
```diff
@@ -128,21 +145,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Find all SAGE variables\n",
+    "### Find all TOPMed DCC Harmonized variables\n",
     "We can search for variables related to our search query using the `bdc::find.in.dictionary` function. \n",
```
Comment (simranmakwana) [inline NHLBI_BioData_Catalyst/R/4_Genomic_Queries.ipynb:149]: I don't think this is the function used any more, isn't this searchPicsure now?

**Review by simranmakwana — CHANGES_REQUESTED** [review]
Main comment: I think the find.in.dictionary function is now searchPicsure?

File: NHLBI_BioData_Catalyst/Rstudio/4_Genomic_Queries.Rmd (hunk 3 above)
Comment (emhug) [reply NHLBI_BioData_Catalyst/Rstudio/4_Genomic_Queries.Rmd:65]: Great catch! thanks!

**Pull request merged**